packages/scm/src/browser/scm-widget.tsx

```diff
@@ -115,7 +118,7 @@ export class ScmWidget extends ScmNavigableListWidget<ScmResource> implements St
         this.scmService.onDidChangeSelectedRepositories(repository => {
             if (repository) {
                 this.selectedRepoUri = repository.provider.rootUri;
-                this.title.label = 'Source Control: ' + repository.provider.contextValue;
+                this.title.label = ScmWidget.LABEL + ': ' + repository.provider.contextValue;
```

Comment:
Should not we update title and rerender the widget when `repository` is `undefined` as well?

Reply:
Fixed

Reply:
> Fixed

@vinokurig  maybe you meant https://github.com/theia-ide/theia/pull/5254/files#r287785296?

Regarding this comment, widget and title don't get updated in `else` case

Reply:
Sorry forgot to push

Reply:
ok, let me know when you push

Reply:
already pushed